packages/devtools/lib/src/framework/framework.dart

```diff
@@ -302,6 +302,13 @@ class ActionsContainer {
   final List<ActionButton> _actions = [];
 
   void addAction(ActionButton action) {
+    for (ActionButton _action in _actions) {
+      if (_action.tooltip == action.tooltip) {
```

Comment:
checking based on identical tooltips isn't robust.
Can you use something cleaner?

Reply:
added Id and used it in the removeAction method as well.